File: 15-03-2024_LinkedList_that_is_Sorted_Alternatingly.cpp
class Solution
{
    public:
    // your task is to complete this function
    void sort(Node **head)
    {
         // Code here
         for(Node* p=*head; p!= NULL ; p=p->next){
            for(Node* q =p->next; q != NULL; q =q->next){
                if(p->data > q->data){
                    swap(p->data, q->data);
                }
            }
        }
    }
};

File: 15-03-2024_LinkedList_that_is_Sorted_Alternatingly.java
class Solution {
    
    public Node sort(Node head){
        Node n1 = head;
        Node n2 = head.next;
        while(n2!=null){
            if(n1.data>n2.data){
                int t=n1.data;
                n1.data=n2.data;
                n2.data=t;
            } 
                n2=n2.next;
        if(n2==null){
                n1=n1.next;
                n2=n1.next;
            }
        }
        return head;
        }


}
